codegen/rest.go

```diff
@@ -225,14 +225,14 @@ end
 {{- $varName := varName $parameter.Name $parameter.Type $parameter.Schema.Ref }}
 {{- $varName := $varName | pascalToSnake }}
 {{- $varComment := varComment $parameter.Name $parameter.Type $parameter.Schema.Ref $parameter.Items.Type }}
-{{- if and (eq $parameter.Name "body") $parameter.Schema.Ref }}
+{{- if and (eq $parameter.In "body") $parameter.Schema.Ref }}
```

Comment:
This change seems to have happened in November 2022. I saw that the other code generators transitioned to the use of "In" instead of "Name".